SPECS-CROSS/gcc-bootstrap_cross/gcc-bootstrap_cross.spec

```diff
@@ -0,0 +1,403 @@
+# Provides:       %%{_cross_name}-g++ = %{version}-%{release}
+
+# %%description    c++
+# This package adds C++ support to the GNU Compiler Collection.
+# It includes support for most of the current C++ specification,
+# including templates and exception handling.
+
+# %%package -n     libstdc++
+# Summary:        GNU C Library
+# Group:          System Environment/Libraries
+# Requires:       %%{_cross_name}-libgcc = %{version}-%{release}
+
+# %%description -n libstdc++
+# This package contains the GCC Standard C++ Library v3, an ongoing project to implement the ISO/IEC 14882:1998 Standard C++ library.
+
+# %%package -n     libstdc++-devel
+# Summary:        GNU C Library
+# Group:          Development/Libraries
+# Requires:       %%{_cross_name}-libstdc++ = %{version}-%{release}
+# Provides:       %%{_cross_name}-libstdc++-static = %{version}-%{release}
+
+# %%description -n libstdc++-devel
+# This is the GNU implementation of the standard C++ libraries.
+# This package includes the headers files and libraries needed for C++ development.
+
+# %%package -n     libgomp
+# Summary:        GNU C Library
+# Group:          System Environment/Libraries
+
+# %%description -n libgomp
+# An implementation of OpenMP for the C, C++, and Fortran 95 compilers in the GNU Compiler Collection.
+
+# %%package -n     libgomp-devel
+# Summary:        Development headers and static library for libgomp
+# Group:          Development/Libraries
+# Requires:       %%{_cross_name}-libgomp = %{version}-%{release}
+
+# %%description -n libgomp-devel
+# An implementation of OpenMP for the C, C++, and Fortran 95 compilers in the GNU Compiler Collection.
+# This package contains development headers and static library for libgomp
+
+%prep
+%setup -q -n gcc-%{version}
+# disable no-pie for gcc binaries
+sed -i '/^NO_PIE_CFLAGS = /s/@NO_PIE_CFLAGS@//' gcc/Makefile.in
+
+%build
+CFLAGS="`echo " %{build_cflags} " | sed 's/-Werror=format-security/-Wno-error=format-security/'`"
+CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/-Werror=format-security/-Wno-error=format-security/'`"
+export CFLAGS
+export CXXFLAGS
+
+export glibcxx_cv_c99_math_cxx98=yes glibcxx_cv_c99_math_cxx11=yes
+SED=sed \
+# Ideally we would like to model this after the %%configure macro in the future.
+./configure \
+            --prefix=%{_cross_prefix} \
+            --target=%{_tuple} \
+            --disable-multilib \
+            --enable-shared \
+            --enable-threads=posix \
+            --enable-__cxa_atexit \
+            --enable-clocale=gnu \
+            --enable-languages=c,c++,fortran \
+            --disable-bootstrap \
+            --enable-linker-build-id \
+            --enable-plugin \
+            --enable-default-pie \
+            --with-sysroot=%{_cross_sysroot} \
+            --with-native-system-header-dir=%{_includedir}
+make %{?_smp_mflags} all-gcc
+
+%install
+make %{?_smp_mflags} DESTDIR=%{buildroot} install-gcc
+
+rm -rf %{buildroot}%{_cross_prefix}%{_infodir}
```

Comment:
Just curious why this `rm -rf infodir` was needed

Reply:
We tend to do this rather than include the info documentation without packages, just keeping the pattern going.

Reply:
Thanks for the explanation.